ChaosMod/Effects/db/Vehs/Vehs30mphLimit.cpp

```diff
@@ -22,7 +22,7 @@ static void OnStop()
 
 static RegisterEffect registerEffect(EFFECT_VEH_SET_TOPSPEED_30MPH, nullptr, OnStop, OnTick, EffectInfo
 	{
-		.Name = "30MPH Speed Limit",
+		.Name = "20MPH Speed Limit",
```

Comment:
This would require renaming everything else containing „30 mph“
Also 30mph is pretty slow enough

Reply:
0xygen, I'm new to coding and also, this is meant to just be a pull request anyone can merge or look at and also, it's meant to make effects kinda extreme. It's kinda me experimenting around with stuff, as well. Also, a lot of stuff is inspired from the original chaos mod. For example, "drawing with alpha 0" was from another effect. I thought you'd have to do that, although I doubted it. Still, did it anyway.

Gravity field causing crashes and lag? Guess what. This is meant to include that and the additional force of the original. Taken from mo_xi.

Reply:
I guess u can criticize it, but yea. It's mostly an experiment.

Not meant to be balanced in the first place.

Reply:
So if you never meant this to be merged then why did you create the PR? why not keep it on your master or branch on your Fork? I mean, this PR can't be just open forever to keep your changes 🤔